superset/config.py

```diff
@@ -499,6 +499,13 @@ class CeleryConfig(object):  # pylint: disable=too-few-public-methods
 # timeout.
 SQLLAB_QUERY_COST_ESTIMATE_TIMEOUT = 10  # seconds
 
+# Flag that controls if limit should be inforced on the create table as queries.
+SQLLAB_CTA_NO_LIMIT = False
+
+# Function accepts username, schema name and sql that will be run e.g.:
```

Comment:
It's not super clear here what this does or why you'd want to use this config element. I had to read a bit of code to understand it.

```
This allows you to define custom logic around the "CREATE TABLE AS" or CTAS feautre in SQL Lab that defines where the target schema should be for a given user.
```
Then add a proper example with type annotation

Reply:
I think we want the database object as a param too as the policy may differ across databases. Also could be good to pass in the user object instead of the username in case someone would want to dig into roles/perms.